dockerfiles/Dockerfile.migraphx

```diff
@@ -5,57 +5,22 @@
 # Dockerfile to run ONNXRuntime with MIGraphX integration
 #--------------------------------------------------------------------------
 
-FROM ubuntu:20.04
+FROM rocm/pytorch:rocm6.0_ubuntu20.04_py3.9_pytorch_2.0.1
```

Comment:
There is a new one:
rocm/pytorch:rocm6.0_ubuntu20.04_py3.9_pytorch_2.1.1

Reply:
Done. Thanks.